scipy-1.0/paper.tex

```diff
@@ -1079,6 +1079,17 @@ \subsection*{Governance}
 \emph{be careful with wording}.
 The Code of Conduct specifies how breaches can be reported to a Code of Conduct Committee, and outlines procedures for the Committee's response. Our Diversity Statement ``welcomes and encourages participation by everyone''.
 
+\subsection*{Maintainers and contributors}
+
+The SciPy project has approximately 15 active maintainers---people who review
+others' contributions, and in general do everything needed to ensure that the
+software and the project move forward. Maintainers are critical to the health
```

Comment:
You can start with the definition of a maintainer and then switch to SciPy specific details. Also approximately 15 is in fact exactly 15 😃

Reply:
Is it like “Then shalt thou count to fifteen, no more–no less. Fifteen shall be the number thou shalt count, and the number of the counting shall be fifteen. Sixteen shalt thou not count, neither count thou fourteen, excepting that thou then proceed to fifteen. Seventeen is right out."

But if it might vary, maybe "The SciPy project typically has between 10 and 20 active maintainers..." if we don't like "approximately 15"?

Reply:
Hahah no I mean there are exactly 15 so no need to "roundify" it but OK I take it back :)